xbmc/network/test/TestWebServer.cpp

```diff
@@ -596,38 +596,39 @@ TEST_F(TestWebServer, CanGetCachedFileWithOlderIfModifiedSince)
 /** @todo Fix these two tests, they keep failing and
  *  we want to enable the test suite on PR
  */
```

Comment:
might want to remove the comment here if the tests are back working.

Reply:
Haha thanks for the hint, I'm an idiot.